.vscode/vscode.h

```diff
@@ -0,0 +1,163 @@
+
+#define __VSCODE_INTELLISENSE__ 1
+#define __BIGGEST_ALIGNMENT__ 1
+#define __CHAR_MAX__ 0x7F
+#define __SCHAR_MAX__ __CHAR_MAX__
+#define __SHRT_MAX__ 0x7FFF
+#define __INT_MAX__ 0x7FFFFFFF
+#define __LONG_MAX__ 0x7FFFFFFFFFFFFFFF
+#define __LONG_LONG_MAX__ __LONG_MAX__
+#define __CHAR_MIN__ -((__CHAR_MAX__)+1)
+#define __SHRT_MIN__ -((__SHRT_MAX__)+1)
+#define __INT_MIN__ -((__INT_MAX__)+1)
+#define __LONG_MIN__ -((__LONG_MAX__)+1)
+#define __LONG_LONG_MIN__ -((__LONG_LONG_MAX__)+1)
+#define __UCHAR_MAX__ 0xFF
+#define __USHRT_MAX__ 0xFFFF
+#define __UINT_MAX__ 0xFFFFFFFF
+#define __ULONG_MAX__ 0xFFFFFFFFFFFFFFFF
+#define __SIZE_MAX__ __ULONG_MAX__
+#define __SSIZE_MAX__ __LONG_MAX__
+#define __UINTPTR_MAX__ __SIZE_MAX__
+#define __INTPTR_MAX__ __SSIZE_MAX__
+#define __WCHAR_MAX__ __UINT_MAX__
+#define __PTRDIFF_MAX__ __UINTPTR_MAX__
+#define __INTMAX_MAX__ __ULONG_MAX__
+#define __SIZEOF_POINTER__ 8
+#define __SIZEOF_SHORT__ 2
+#define __SIZEOF_INT__ 4
+#define __SIZEOF_LONG__ 8
+#define __SIZEOF_LONG_LONG__ 8
+#define __SIZEOF_PTRDIFF_T__ 8
+#define __SIZEOF_SIZE_T__ 8
+#define __SIZEOF_WCHAR_T__ 4
+#define __SIZEOF_WINT_T__ 4
+#define __SIZEOF_FLOAT__ 4
+#define __SIZEOF_FLOAT128__ 16
+#define __SIZEOF_DOUBLE__ 8
+#define __SIZEOF_FLOAT80__ 10
+#define __SIZEOF_LONG_DOUBLE__ 16
+#define __FLT_MAX__ 1
+#define __DBL_MAX__ 1
+
+#define __INT_LEAST8_TYPE__ signed char;
```

Comment:
Sorry, that was an accident. I was fairly hasty with a lot of these as there were *so many* files to trudge through. I’ll fix it along with the other things.

Reply:
No reason to apologize. That's what code review is for.

I'm really happy with this change, since it's going to go a long way towards helping us attract developers. I want Cosmo to have the best VSCode support possible and I'm committed to maintaining it. I hope the edits earlier today are of assistance.